haas/cli.py

```diff
@@ -684,24 +728,25 @@ def list_nodes(is_free):
     <is_free> may be either "all" or "free", and determines whether
         to list all nodes or all free nodes.
     """
-    if is_free not in ('all', 'free'):
-        raise TypeError("is_free must be either 'all' or 'free'")
-    url = object_url('nodes', is_free)
-    do_get(url)
-
+    q = C.node.list(is_free)
```

Comment:
@zenhack I find it more appropriate to make it Boolean. What do you think ? Like if `is_free` is `True` show only free nodes else show all. But that needs to be changed in API call too. I am not sure if that is out of the context of the PR.  Let me know what you think.